TriviaBot-Question-Submission-Advice.md

```diff
@@ -1,27 +1,27 @@
+Please note that if you submit hints with your question we will remove any hints that look like they mimic our automatic hints, e.g. scrambled answers, pig latin, etc. This is because your own hint may not translate to the other languages that TriviaBot supports. Custom hints are for hints such as, if the question was "what fruit is yellow, usually curved and has a thick skin which is peeled", a good hint would be "actually grows in a bush not on a tree" and not for example "Scrambled: ananab".
 
 **If you are submitting images alongside your question or answer_, you should also follow the [image submission guidelines](https://github.com/brainboxdotcc/triviabot/blob/master/TriviaBot-Image-Submission-Guidelines.md) to ensure your question is reviewed and approved swiftly.**
 
-Questions should be submitted in English, unless you don't speak fluent English in which case feel free to submit questions in your native tongue and we will attempt to translate it.
+Questions should be submitted in English, unless you don't speak fluent English in which case feel free to submit questions in your native tongue and we will attempt to translate it. Not all questions that are submitted in different languages will be approved.
 
 Duplicate questions will be rejected, but do not be disheartened if your question is rejected on these grounds, this means it was a **good question**, we just already know it. Try something else about the same subject that we may not have in the bot yet!
 
 _We will not accept any questions which clearly have a small amount of time or research put into them (a.k.a. `spam questions`) and/or are hard for moderators to verify. Here are some examples of low quality questions that will get denied and possibly a submission ban or warn for the submitter._
 
 ## Examples of bad questions, and why they are bad questions
 
-`what is 1 + 1`:
+`What is 1 + 1?`:
 This question is of low quality because it incorporates something that almost everyone knows and isn't trivia. We will deny this question.
 
 `Is <something> <something>?`:
 This question is of low quality because it is a simple yes/no question that can be answered in under a second by just typing `yes and no` in chat.
 
 `Who is the current president of the USA`:
-This question is of low quality since it doesn't provide a year and just says `current`. This means that in x years this question will have to be replaced. We will deny this question. Any question that will not be valid in 10 years will be denied.
+This question is of low quality since it doesn't provide a year and just says `current`. This means that in x years this question will have to be replaced or alternitvly you could say `Who is the president of the USA in 2021`. We will deny this question. Any question that will not be valid in 10 years will be denied. 
```

Comment:
no need for example, as the line then says we will deny a question asking who the president was in 2021...